internal/cmd/fetcher/main.go

```diff
@@ -292,7 +292,7 @@ func copyFile(
 	}()
 	filename := filepath.Base(dest)
 	switch filename {
-	case "Dockerfile", "Dockerfile.wasm", "buf.plugin.yaml", "package.json":
+	case "Dockerfile", "Dockerfile.wasm", "buf.plugin.yaml", "build.csproj", "package.json":
```

Comment:
This can be tweaked more later, but this at least will unblock the automatic fetch versions workflow for C# plugins.